src/include/swap.h

```diff
@@ -47,6 +47,7 @@ static inline uint64_t
 __wt_bswap64(uint64_t v)
 {
 	return (
+	    /* NOLINTNEXTLINE(misc-redundant-expression) */
 	    ((v << 56) & 0xff00000000000000UL) |
```

Comment:
Clang Tidy seems to think that this line doesn't do anything. The code looks sane to me so I've just suppressed the linter here.